front/src/modules/ui/data/data-table/components/DataTable.tsx

```diff
@@ -118,8 +127,20 @@ export const DataTable = ({ updateEntityMutation }: DataTableProps) => {
     },
   });
 
+  const tableRowIds = useRecoilValue(tableRowIdsState);
+  // const visibleTableColumns = useRecoilScopedValue(
+  //   visibleTableColumnsScopedSelector,
+  //   TableRecoilScopeContext,
+  // );
+
+  const defaultOnClick = () => {
```

Comment:
Prefer allowing undefined in EmptyTable onClick method. And check if onClick is defined in EmptyTable